### PR TITLE
fix: TriggerHotkeyByKeySequenceRequest wrong parameter

### PR DESCRIPTION
**Pull request by Pjiesco**

### CODE REVIEW

File: src/main/java/net/twasi/obsremotejava/model/KeyModifiers.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+@Setter(AccessLevel.PRIVATE)
+@ToString
+@NoArgsConstructor
+public class KeyModifiers {
+    private boolean shift;
+    private boolean alt;
+    private boolean control;
+    private boolean command;
+
+    public static KeyModifiers fromTypeList(List<KeyModifierType> modifierList) {
```
Comment (ChristopheCVB) [inline src/main/java/net/twasi/obsremotejava/model/KeyModifiers.java:19]: After thought, I'm not a fan of this extra layer.
Maybe we could simply add the Builder annotation to this and ask the developer an instance in https://github.com/Twasi/websocket-obs-java/pull/56/files#diff-2c09711e485dceba4478a51b66c08dc9d89c8ad2c2ced472da35b0041a4e7ac2R16

**Review by ChristopheCVB — CHANGES_REQUESTED** [review]
Not fan of the extra layer

File: src/main/java/net/twasi/obsremotejava/model/KeyModifiers.java (hunk 1 above)
Comment (Pjiesco) [reply src/main/java/net/twasi/obsremotejava/model/KeyModifiers.java:19]: Yeah that could indeed be better but I also like the idea of this. But I will remove this and simply add the builder.

**Review by ChristopheCVB — CHANGES_REQUESTED** [review]
Rebase your branch with `5.x.x/develop`

File: src/main/java/net/twasi/obsremotejava/model/KeyModifiers.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package net.twasi.obsremotejava.model;
```
Comment (ChristopheCVB) [inline src/main/java/net/twasi/obsremotejava/model/KeyModifiers.java:1]: Move this class to `TriggerHotkeyByKeySequenceRequest.KeyModifiers`

File: src/main/java/net/twasi/obsremotejava/message/request/general/TriggerHotkeyByKeySequenceRequest.java [hunk 3]
```diff
@@ -13,7 +13,7 @@
 public class TriggerHotkeyByKeySequenceRequest extends Request {
     private final Data requestData;
 
-    public TriggerHotkeyByKeySequenceRequest(String keyId, List<String> keyModifiers) {
+    public TriggerHotkeyByKeySequenceRequest(String keyId, KeyModifiers keyModifiers) {
```
Comment (ChristopheCVB) [inline src/main/java/net/twasi/obsremotejava/message/request/general/TriggerHotkeyByKeySequenceRequest.java:16]: Just keep this constructor, remove others

**Review by ChristopheCVB — CHANGES_REQUESTED** [review]
Rebsae and check comments

**Pull request closed without merging**